velox/functions/prestosql/Bitwise.h

```diff
@@ -28,6 +30,29 @@ struct BitwiseAndFunction {
   }
 };
 
+template <typename T>
+struct BitCountFunction {
+  template <typename TInput>
+  FOLLY_ALWAYS_INLINE bool call(int64_t& result, TInput num, TInput bits) {
+    if (bits == kMaxBits) {
+      result = std::bitset<kMaxBits>(num).count();
+      return true;
+    }
+    if (bits <= 1 || bits > kMaxBits) {
```

Comment:
Out-of-range values of bits should produce an error:

```
presto:di> select bit_count(5, 0);
Query 20220201_142908_35354_t9upv failed: Bits specified in bit_count must be between 2 and 64, got 0

```

Reply:
I had a discussing on this with @aditi-pandit for out of range values returning false will default to NULL. Since this could be called for individual rows, crashing the entire query may not be the right thing. May be I should be add this to the documentation as well.

Example:

```

```